.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "files.insertFinalNewline": true
+}
```

Comment:
Been thinking on this a bit, for this project please just don't commit the .vscode directory
If you want to add it to the .gitignore that's fine